packages/smooth_app/lib/pages/scan/search_page.dart

```diff
@@ -19,7 +19,8 @@ void _performSearch(
   String query, {
   EditProductQueryCallback? editProductQueryCallback,
 }) {
-  if (query.trim().isEmpty) {
+  query = query.trim();
```

Comment:
Just curious , we aren't handling the "redbull" and "Redbull"  as two different things here

Reply:
@AshAman999 The fix was only about spaces, not lower / upper case.
"redbull" and "Redbull" will be considered as different.

Reply:
So we're going to handle that case or leave it as it is for now ?

Reply:
@AshAman999 I think "redbull" and "Redbull", both should be treated as separate keywords as on UI side they don't look same (also There could be some brands having capital keywords?).

In search history, "redbull" and "  redbull  " were treated as separate keywords but to user they would look same (because couldn't tell difference between whitespaces), hence duplicate.

Reply:
This pops a question in my mind that should there be a some sort of suggestion in search box ? if user inputs something like "REdbUll" or "r ed  bul  l", then should there be suggestion for correct word?

Maybe this could be a feature?

Reply:
Yeah some kind of local autocomplete would make sense. Though as far as I can remember we only store 5 or so last searches atm.

Reply:
If you guys have suggestions beyond this PR feel free to open a new issue, so that we can merge this successful PR first.
Seeing several times the exact same text (e.g. "redbull") (modulo the spaces) was a bit problematic, and that's what the PR was about. Done.